src/Campaigns/Blocks/CampaignDonors/render.php

```diff
@@ -40,7 +39,7 @@
         'donorName.meta_value as name'
     )
         ->groupBy('donorIdMeta.meta_value')
-        ->orderBy('amount', 'DESC');
+        ->orderByRaw('CAST(amountMeta.meta_value AS DECIMAL) DESC');
```

Comment:
At this point, we are not using `amountMeta.meta_value` directly. Our query needs to be ordered by `amount`, as it is an alias representing the sum of all values from the same donor.

![CleanShot 2025-02-28 at 13 59 37](https://github.com/user-attachments/assets/5a7b03f9-c111-49a6-b948-7b007f5b50c8)
```php
->orderByRaw('CAST(amountMeta.meta_value as DECIMAL) DESC');
```

![image](https://github.com/user-attachments/assets/742ba2e0-df23-45d3-b34d-8bac04ed0c5d)
```php
->orderByRaw('CAST(amount as DECIMAL) DESC');
```

Reply:
Nice catch! Resolved in 89e82d214b51443d3ebc858487c147f13b183e01